addons/website/static/tests/tours/snippet_images_wall.js

```diff
@@ -0,0 +1,46 @@
+odoo.define("website.tour.snippet_images_wall", function (require) {
+"use strict";
+
+const tour = require("web_tour.tour");
+const wTourUtils = require("website.tour_utils");
+
+tour.register("snippet_images_wall", {
+    test: true,
+    url: "/",
+}, [
+    ...wTourUtils.clickOnEditAndWaitEditMode(),
```

Comment:
I would remove the `...wTourUtils.clickOnEditAndWaitEditMode()` and the `const tour = require("web_tour.tour");` and use the `wTourUtils.registerWebsitePreviewTour(` with `edition: true`

Reply:
Oups this does not exits yet in 15.0 MB.

Reply:
What about in 32MB?

Reply:
I meant "my bad" :sweat_smile: